src/EventSubscriber/WebformSubmissionDataEventSubscriber.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Drupal\os2forms_rest_api\EventSubscriber;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\file\FileInterface;
+use Drupal\webform\WebformInterface;
+use Drupal\webform_rest\Event\WebformSubmissionDataEvent;
+use Psr\Log\LoggerAwareTrait;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * WebformSubmissionEventSubscriber, for updating Webform Submission GET data.
+ */
+class WebformSubmissionDataEventSubscriber implements EventSubscriberInterface {
+  use LoggerAwareTrait;
+
+  /**
+   * Entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  private EntityTypeManagerInterface $entityTypeManager;
+
+  /**
+   * Map from entity type to webform element types.
+   */
+  private const LINKED_ELEMENT_TYPES = [
+    'file' => [
+      'webform_image_file',
+      'webform_document_file',
+      'webform_video_file',
+      'webform_audio_file',
+      'managed_file',
+    ],
+  ];
+
+  /**
+   * Constructor.
+   */
+  public function __construct(EntityTypeManagerInterface $entityTypeManager, LoggerInterface $logger) {
+    $this->entityTypeManager = $entityTypeManager;
+    $this->setLogger($logger);
+  }
+
+  /**
+   * Event handler.
+   */
+  public function onWebformSubmissionDataEvent(WebformSubmissionDataEvent $event): void {
+    $linkedData = $this->buildLinked($event->getWebformSubmission()->getWebform(), $event->getData());
+
+    if (!empty($linkedData)) {
+      $event->setData($event->getData() + ['linked' => $linkedData]);
+    }
+  }
+
+  /**
+   * Builds linked entity data.
+   *
+   * @see https://support.deskpro.com/en/guides/developers/deskpro-api/basics/sideloading
```

Comment:
It seems the url has switched to [sideloading](https://support.deskpro.com/en-US/guides/developers/deskpro-api/basics/sideloading)